### ISSUE TITLE
Support for couchbase in java APM

### ISSUE THREAD

**Issue by brentm5** [DataDog/dd-trace-java#412]
Couchbase includes memcache functionality, traditional nosql, and an advanced query language so there are a several different buckets this fits into APM perspective.  

The link to the offical client is here: https://github.com/couchbase/couchbase-java-client

**Comment by tylerbenson** [comment]
@brentm5 I added this in #464.  Can you try out the latest snapshot build here: https://oss.jfrog.org/artifactory/oss-snapshot-local/com/datadoghq/dd-java-agent/0.15.0-SNAPSHOT/dd-java-agent-0.15.0-SNAPSHOT.jar

Thanks!

**Comment by brentm5** [comment]
It would also be nice to have the bucket name included in the metrics, for example

```
metric: trace.couchbase.call.hits
tag: resource_name: Bucket.get
tag: bucket: bucket1
```

**Comment by tylerbenson** [comment]
@brentm5 I've implemented your first suggestion in #495.  The second, I'll have to review with some other folks on my team.  Thanks for the feedback.
